dist/vokal-ng-api.js

```diff
@@ -0,0 +1,365 @@
+(function (root, factory) {
```

Comment:
Yeah, that whole chunk at the top is what grunt-umd creates, which is why it is in dist but not source